pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -28,19 +28,20 @@ func (f *frontend) putOrPatchOpenShiftCluster(w http.ResponseWriter, r *http.Req
 	ctx := r.Context()
 	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
 	vars := mux.Vars(r)
+	installVersion := vars["install-version"]
```

Comment:
install version shouldn't be coming from vars, it should be in the body of the payload